spec/controllers/api/users_controller_spec.rb

```diff
@@ -1,74 +0,0 @@
-RSpec.describe Api::UsersController do
-  before { sign_in(user) }
-
-  let(:user) { users(:user) }
-
-  describe '#update' do
-    subject(:patch_update) { patch(:update, params:) }
-
-    context 'when a user is updating themself' do
-      let(:user_id_param) { user.id }
-
-      context 'when the params are valid' do
-        let(:valid_params) { { id: user_id_param, user: { preferences: { color: 'blue' } } } }
-        let(:params) { valid_params }
-
-        it 'returns a 200 status code' do
-          patch_update
-          expect(response).to have_http_status(200)
-        end
-
-        it 'updates the user' do
-          expect { patch_update }.to change { user.reload.preferences }.to({ 'color' => 'blue' })
-        end
-
-        it 'responds with the user as JSON' do
-          patch_update
-          user.reload
-
-          expect(response.parsed_body).to include(
-            'email' => user.email,
-            'id' => user.id,
-            'preferences' => { 'color' => 'blue' },
-          )
-        end
-      end
-
-      context 'when updating preferences' do
-        let(:preferences) { { favorite_number: 2 } }
-        let(:params) { { id: user_id_param, user: { preferences: } } }
-
-        it 'updates the user' do
-          expect { patch_update }.
-            to change { user.reload.preferences }.
-            to(hash_including(preferences.stringify_keys))
-        end
-      end
-    end
-
-    context 'when a user attempts to update another user' do
```

Comment:
It's no longer really possible/relevant to test this, since the new singular `resource` route doesn't have a user id param.